client/my-sites/upgrades/checkout-thank-you/jetpack-thank-you-card.jsx

```diff
@@ -109,15 +158,27 @@ class JetpackThankYouCard extends Component {
 
 	componentDidUpdate() {
 		const site = this.props.selectedSite;
-		if ( site &&
-			site.jetpack &&
-			site.canUpdateFiles &&
-			site.canManage() &&
-			this.allPluginsHaveWporgData() &&
-			! this.props.isInstalling &&
-			this.props.nextPlugin
+		const { plugins } = this.props;
+
+		if ( ! site ||
+			! site.jetpack ||
+			! site.canUpdateFiles ||
+			! site.canManage() ||
```

Comment:
Yessir. In this case, we are using the redux site instead of sites list, we're then passing that redux site `lib/site/jetpack`.